cdc/sink/mq/mq_test.go

```diff
@@ -70,16 +92,15 @@ func (s mqSinkSuite) TestKafkaSink(c *check.C) {
 	}()
 
 	sink, err := NewKafkaSaramaSink(ctx, sinkURI, fr, replicaConfig, opts, errCh)
-	c.Assert(err, check.IsNil)
+	require.Nil(t, err)
 
 	encoder := sink.encoderBuilder.Build()
 
-	c.Assert(encoder, check.FitsTypeOf, &codec.JSONEventBatchEncoder{})
-	c.Assert(encoder.(*codec.JSONEventBatchEncoder).GetMaxBatchSize(), check.Equals, 1)
-	c.Assert(encoder.(*codec.JSONEventBatchEncoder).GetMaxMessageBytes(), check.Equals, 1048576)
+	// c.Assert(encoder, check.FitsTypeOf, &codec.JSONEventBatchEncoder{})
```

Comment:
Why we need to comment this?

Reply:
That is an oversight, thanks for your review.